.ci/pitest.sh

```diff
@@ -36,12 +36,20 @@ pitest-annotation|pitest-design \
 |pitest-sizes|pitest-whitespace \
 |pitest-api \
 |pitest-packagenamesloader \
-|pitest-common-2|pitest-misc|pitest-xpath)
+|pitest-common-2|pitest-xpath)
   mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
   declare -a ignoredItems=();
   checkPitestReport "${ignoredItems[@]}"
   ;;
 
+pitest-misc)
+  mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
+  declare -a ignoredItems=(
+  "OrderedPropertiesCheck.java.html:<td class='covered'><pre><span  class='survived'>            if (!prevKeyFound &#38;&#38; prevKeyPattern.matcher(line).matches()) {</span></pre></td></tr>"
```

Comment:
We prefer no surviving mutants.
I will look into this and try to help.

Reply:
The code can be changed to `if (true && prevKeyPattern.matcher(line).matches()) {` and no tests are failing. So you need a test to prove that the code can't be changed.

Unless the code changes, the easiest way to remove this mutation is to make the code like:
`````
if (prevKeyFound) {
...
} else {
...
}
`````

Reply:
I will look into it.
The `if(true &&` trick does not look good and will not solve the problem.
Because 1) 
`[checkstyle] [ERROR] checkstyle/src/main/java/com/puppycrawl/tools/checkstyle/checks/OrderedPropertiesCheck.java:152:31: Expression can be simplified. [SimplifyBooleanExpression]`

and 2) the mutation problem persists.

At the `if (prevKeyFound)` suggestion I am still working. The test coverage is not 100% and mutation score is 99 -> not 100. So here the (local) build failes.

Reply:
@thomassenger If you misread what I said. The `if(true` is what pitest is doing behind the scenes, not what I wanted you to change the code to.
The code can be changed to `if (prevKeyFound) { } else { }` to remove the mutation.

Reply:
I changed the code to `if (prevKeyFound) { } else { }` now we will not have timed out mutations. But PIT will try to count backward through the file and change all if to else.
The code looks not very good. I do not know if it is a good strategy to seek for the previous and current key.
The testLineNumbers() is also not removed yet.

Reply:
I found a different solution.
Done.